lib/oxidized/jobs.rb

```diff
@@ -44,14 +44,26 @@ def new_count
       @want = @max if @want > @max
     end
 
+    def increment
+      # Increments the job count if safe to do so, which means:
+      # a) less threads running than the total amount of nodes
+      # b) we want less than the max specified number of threads
+
+      return unless (@want < @nodes.size) && (@want < @max)
```

Comment:
What do you think of @want < [@nodes.size, @max].min?

Reply:
Or [@want, @nodes.size, @max].min == @want

Reply:
Can actually summarize the whole function into:

`want = [(@want + 1), @nodes.size, @max].min`

Reply:
If you think it's best. I didn't suggest the option to reduce code size, I suggested it because it takes longer for me to parse and understad condition1 && condition2, than [a,b,c].min == a, but this is just personal opion.

I'm happy to accept the PR as is, but if you want to change, that's also fine.

Reply:
@ytti I could go either way (separate conditional vs. inline with the assignment); your call here.

Both are functional equivalent and I don't find one easier/harder to interpret personally.